internal/cortex/frontend/transport/handler.go

```diff
@@ -156,7 +150,7 @@ func (f *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	// Check if caching is enabled.
 	if f.lruCache != nil {
 		// Store query expression.
-		queryExpressionNormalized = f.regex.ReplaceAllString(r.URL.Query().Get("query"), " ")
+		queryExpressionNormalized = regex.ReplaceAllString(r.URL.Query().Get("query"), " ")
```

Comment:
where did the `regex` variable comes from?

Reply:
from utils.go